astropy/stats/lombscargle/core.py

```diff
@@ -191,6 +197,9 @@ def autofrequency(self, samples_per_peak=5, nyquist_factor=5,
         maximum_frequency : float (optional)
             If specified, then use this maximum frequency rather than one
             chosen based on the average nyquist frequency.
+        return_freq_limits : bool (optional)
```

Comment:
We usually don't put "optional" in parens, but simply prepend with a comma (again based on the numpydoc style), e.g. :  `return_freq_limits : bool, optional`.  I do see several other instances of the `(optional)` usage in astropy, but most of them are in the stats modules that I think you wrote :smile:  (e.g. `bayesian_blocks.py` and `histogram.py`).  Leave it for now.